ruma-events/benches/event_deserialize.rs

```diff
@@ -0,0 +1,109 @@
+// `cargo bench` works but, if you use `cargo bench -- --save-baseline <name>`
```

Comment:
I don't know whether I suggested the comma at that position, but it's still wrong..